src/intl/translations/de.ts

```diff
@@ -0,0 +1,52 @@
+export const de = {
+    locale: 'de',
+    powered_by_gitbook: 'Bereitgestellt von GitBook',
+    switch_to_dark_theme: 'Zum dunklen Modus wechseln',
+    switch_to_light_theme: 'Zum hellen Modus wechseln',
+    switch_to_system_theme: 'Zum Systemmodus wechseln',
+    search: 'Suche',
+    search_or_ask: 'Fragen oder Suchen',
+    search_input_placeholder: 'Inhalt durchsuchen',
+    search_ask_input_placeholder: 'Inhalt durchsuchen oder eine Frage stellen',
+    search_no_results: 'Keine Ergebnisse für "${1}".',
+    search_scope_space: 'Nur in ${1}',
+    search_scope_all: 'Alle Inhalte',
+    search_ask: 'Fragen "${1}"',
+    search_ask_sources: 'Quellen',
+    search_ask_no_answer: 'Es konnte keine Antwort auf Ihre Frage gefunden werden, versuchen Sie es mit einer anderen Frage.',
+    search_ask_error: 'Etwas ist schief gelaufen. Bitte versuchen Sie es später noch einmal.',
+    on_this_page: 'Auf dieser Seite',
+    next_page: 'Nächste',
+    previous_page: 'Vorherige',
+    page_last_modified: 'Zuletzt aktualisiert ${1}',
+    was_this_helpful: 'War das hilfreich?',
+    was_this_helpful_positive: 'Ja, das war es!',
+    was_this_helpful_neutral: 'Nicht sicher',
+    was_this_helpful_negative: 'Nein',
+    was_this_helpful_thank_you: 'Danke!',
+    annotation_button_label: 'Kommentar öffnen',
+    code_copied: 'Kopiert!',
+    code_copy: 'Kopieren',
+    table_of_contents_button_label: 'Inhaltsverzeichnis öffnen',
+    cookies_title: 'Cookies',
+    cookies_prompt:
+        'Diese Website verwendet Cookies, um ihre Dienste bereitzustellen und den Datenverkehr zu analysieren. Durch die Nutzung dieser Website akzeptieren Sie die ${1}.',
+    cookies_prompt_privacy: 'Datenschutzrichtlinie',
+    cookies_accept: 'Akzeptieren',
+    cookies_reject: 'Ablehnen',
+    cookies_close: 'Schließen',
+    edit_on_git: 'Bearbeiten auf ${1}',
+    notfound_title: 'Seite nicht gefunden',
+    notfound: 'Die Seite, die Sie suchen, existiert nicht.',
```

Comment:
I was wondering about a simpler one like "Die gesuchte Seite existiert nicht"?

Reply:
Good suggestion, changed it!